site/content/en/docs/drivers/hyperkit.md

```diff
@@ -29,6 +29,35 @@ If you are using `dnsmasq` and `minikube` fails, add `listen-address=192.168.64.
 
 If you are running other DNS servers, shut them off or specify an alternative bind address.
 
+### Hyperkit upgrade version
+The version of correct Hyperkit not being in the path could cause issues in starting minikube.
+Try upgrading Hyperkit and ensure the upgraded version is in the path.
+
+The latest version of Hyperkit could be found [here](https://github.com/moby/hyperkit/releases).
+
+You can check your current version by:
+
+```
+hyperkit -v
+```
+
+To upgrade try:
```

Comment:
This assumes brew is installed. Is it possible to provide non-brew instructions as well?

Reply:
we have instatllation sections for hyperkit https://minikube.sigs.k8s.io/docs/drivers/hyperkit/#hyperkit-installation
there we say either brew or compile it from source.